src/pages/CartPage.vue

```diff
@@ -24,10 +26,17 @@ import CartConfirm from '/@/components/Cart/CartConfirm.vue'
 
 const cartStore = useCart()
 const router = useRouter()
-useTitle(ref('まとめて借りる'))
 
 const itemCount = computed(() => cartStore.cart.length)
 
+const title = '貸し出し手続き'
```

Comment:
```suggestion
const title = '借りる'
```
とかですかね
`貸し出し手続き`はちょっとbooQの文言としては固すぎな気がします(わざわざページタイトルを変えなくてもいいっていうのは同意です)